src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
The best solution is to insert the image into the <img> tag. This way we will be more consistent with semantics.

```suggestion
      <img class="header__logo" alt="logo"/>
```